src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -141,3 +146,12 @@ public boolean equals(Object other) {
                         && this.internalList.equals(((UniquePersonList) other).internalList));
     }
 }
+
+// List of comparators to prepare for future enhancements
+// Comparator to compare name
+class NameComparator implements Comparator<Person>{
```

Comment:
Good job in creating the comparator class, another method that you could do is the in-line method where you don't have to write an additional class.

`Collections.sort(internalList, new Comparator<Person>() { ... });`

and the ... is replaced with the same code (compare) is doing.